tuxemon/core/components/config.py

```diff
@@ -51,6 +51,11 @@ def __init__(self, file="tuxemon.cfg"):
         self.resolution_y = self.config.get("display", "resolution_y")
         self.resolution = (int(self.resolution_x), int(self.resolution_y))
 
+        self.sound_volume = float(self.config.get("sound", "sound_volume"))
```

Comment:
Now that the game is becoming more and more playable. I'm wondering if we should start to include some logic for "upgrading" a user's `tuxemon.cfg` if some sections are missing. When I tried running, it crashes because I have an older config file in my home directory:

```python
Traceback (most recent call last):
  File "tuxemon.py", line 60, in <module>
    from core.main import main
  File "/tmp/Tuxemon/tuxemon/core/main.py", line 33, in <module>
    from . import prepare
  File "/tmp/Tuxemon/tuxemon/core/prepare.py", line 77, in <module>
    CONFIG = config.Config(CONFIG_FILE_PATH)
  File "/tmp/Tuxemon/tuxemon/core/components/config.py", line 54, in __init__
    self.sound_volume = float(self.config.get("sound", "sound_volume"))
  File "/usr/lib/python2.7/ConfigParser.py", line 607, in get
    raise NoSectionError(section)
ConfigParser.NoSectionError: No section: 'sound'
```

Easy fix for this is to just remove the old the old config or add the new section, but this issue will invariably be posted by users who are upgrading.